ui/src/app/shared/services/banner_ui.ts

```diff
@@ -0,0 +1,9 @@
+import {BannerMessage} from '../models';
+import requests from './requests';
+
+const apiUrl = 'https://api.mocki.io/v1/b837272d';
```

Comment:
mocki.io is an api mocking service.  I assume this apiUrl will be replaced with a real url before this PR is merged?

Reply:
Yes @jopit , API calls that argocd use are expensive so we will have to figure out a way to recieve data.